spec/mailers/rents_mailer_spec.rb

```diff
@@ -0,0 +1,26 @@
+require 'rails_helper'
+
+RSpec.describe RentsMailer, type: :mailer do
+  include_context 'User, Book and Rent'
+
+  let(:user_mail) { create(:user, email: 'juan@gmail.com') }
+  let(:rent) { create(:rent, user_id: user_mail.id, book_id: book.id) }
+
+  describe 'rent_confirmation' do
+    let(:mail) { RentsMailer.rent_confirmation(rent) }
+
+    it 'check the headers' do
+      expect(mail.subject).to eq('[NO-REPLY] Details about the book rental!')
+      expect(mail.from).to eq(['postmaster@sandboxd23676cf8ca74d14b04278ed1d309c95.mailgun.org'])
```

Comment:
This kind of data should be in your env file

Reply:
Okay! Now a variable from the credentials is called